js/ui/speed_dial_action/speed_dial_action.js

```diff
@@ -91,6 +90,11 @@ const SpeedDialAction = Widget.inherit({
             case "icon":
                 initAction(this);
                 break;
+            case "visible":
```

Comment:
What about this scenario:
1. Create SDA with `visible:false`
2. Change `icon` option
It seems that this https://github.com/DevExpress/DevExtreme/pull/8919/files#diff-645315b92c27d92aebe52d8df754b0e0R91 initAction will fire. Or not?

Reply:
Add test and fix